src/style-banner.scss

```diff
@@ -163,6 +166,46 @@
   }
 }
 
+// =-=-=-=-= roundedBorder -=-=-=-=-=-=
+
+:host([roundedBorder]),
```

Comment:
I don't have a great way to enforce this because SassLint is a dead project, but I still subscribe to many of their style guide rules. 

In this case, please review https://github.com/sasstools/sass-lint/blob/master/docs/rules/empty-line-between-blocks.md

This should be consistent across all Auro projects, for example https://github.com/AlaskaAirlines/auro-accordion/blob/master/src/style.scss

Reply:
👍 Yeah the intention is good here and I have no problem adding extra spaces.  This type of thing is generally to help the developer experience and make the code easier to read.  Though I would argue that these type of rules/opinions only help the developer experience when they can be automated otherwise they become tedious, and hard to enforce especially as we have more people contributing or in charge of handing PR's.

Reply:
Trust me I know. I would LOVE to use a linter, but SassLint is a mess right now.